msteams-platform/sbs-meeting-content-bubble.yml

```diff
@@ -89,6 +89,7 @@ items:
        You can also create a new resource group (select **Create new** > enter resource name > select **OK**).
     
     1. If you've created a new resource group, select the required location from **New resource group location** dropdown list.
```

Comment:
Link to create resource group is not required here, Line 89 already covers this step and there is a link in the UI. Remove all similar instances of the link from other sbs guides

Reply:
The links are removed as suggested..